src/SFA.DAS.Funding.SystemAcceptanceTests/StepDefinitions/Fm36StepDefinitions.cs

```diff
@@ -50,332 +54,723 @@ public async Task Fm36DataExists()
-            .Should().OnlyContain(x => x.Value == (instalmentAmount * 95) / 100, $"Not all {LearningDeliveryPeriodisedValuesAttributeNames.ProgrammeAimProgFundIndMinCoInvest} are {(instalmentAmount * 95) / 100}.");
+                .GetValuesForAttribute(
+                    LearningDeliveryPeriodisedValuesAttributeNames.ProgrammeAimProgFundIndMinCoInvest)
+                .Should().OnlyContain(x => x.Value == (instalmentAmount * 95) / 100,
+                    $"Not all {LearningDeliveryPeriodisedValuesAttributeNames.ProgrammeAimProgFundIndMinCoInvest} are {(instalmentAmount * 95) / 100}.");
 
             fm36Learner.LearningDeliveries.FirstOrDefault()?.LearningDeliveryPeriodisedValues
-            .GetValuesForAttribute(LearningDeliveryPeriodisedValuesAttributeNames.ProgrammeAimTotProgFund)
-            .Should().OnlyContain(x => x.Value == instalmentAmount, $"Not all {LearningDeliveryPeriodisedValuesAttributeNames.ProgrammeAimTotProgFund} are {instalmentAmount}.");
+                .GetValuesForAttribute(LearningDeliveryPeriodisedValuesAttributeNames.ProgrammeAimTotProgFund)
+                .Should().OnlyContain(x => x.Value == instalmentAmount,
+                    $"Not all {LearningDeliveryPeriodisedValuesAttributeNames.ProgrammeAimTotProgFund} are {instalmentAmount}.");
 
             fm36Learner.LearningDeliveries.FirstOrDefault()?.LearningDeliveryPeriodisedTextValues
-            .GetValuesForAttribute(LearningDeliveryPeriodisedTextValuesAttributeNames.FundLineType)
-            .Should().OnlyContain(x => x.Value.ToString() == "16-18 Apprenticeship (Employer on App Service)", $"Not all {LearningDeliveryPeriodisedTextValuesAttributeNames.FundLineType} are 16-18 Apprenticeship (Employer on App Service).");
+                .GetValuesForAttribute(LearningDeliveryPeriodisedTextValuesAttributeNames.FundLineType)
+                .Should().OnlyContain(x => x.Value.ToString() == "16-18 Apprenticeship (Employer on App Service)",
+                    $"Not all {LearningDeliveryPeriodisedTextValuesAttributeNames.FundLineType} are 16-18 Apprenticeship (Employer on App Service).");
 
             fm36Learner.LearningDeliveries.FirstOrDefault()?.LearningDeliveryPeriodisedTextValues
-            .GetValuesForAttribute(LearningDeliveryPeriodisedTextValuesAttributeNames.LearnDelContType)
-            .Should().OnlyContain(x => x.Value.ToString() == "Contract for services with the employer", $"Not all {LearningDeliveryPeriodisedTextValuesAttributeNames.LearnDelContType} are \"Contract for services with the employer\".");
+                .GetValuesForAttribute(LearningDeliveryPeriodisedTextValuesAttributeNames.LearnDelContType)
+                .Should().OnlyContain(x => x.Value.ToString() == "Contract for services with the employer",
+                    $"Not all {LearningDeliveryPeriodisedTextValuesAttributeNames.LearnDelContType} are \"Contract for services with the employer\".");
```

Comment:
Is this aggressive newlining intentional? Personally I think it looks cleaner with the simple asserts on one line and these later ones as:

![image](https://github.com/user-attachments/assets/0421c585-94d6-4a43-88bf-87d81e21344a)